app/view/show.py

```diff
@@ -11,8 +11,33 @@
 from app.pacman import get_pkg
 from app.util import chunks, multiline_to_list
 from collections import defaultdict
+from urllib.parse import urlencode
 
 
+def link_encode(url, query):
+    query = {k: v for k, v in query.items()}
+    data = urlencode(query)
+    return "%s?%s" % (url, data)
```

Comment:
Please always use the "{}".format() style. I like my bike shed pink and consistent.